### ISSUE TITLE
View to include proper id for assistive technologies

### ISSUE THREAD

**Issue by abrassat** [xwikisas/application-task#336]
**Describe the bug**
The task properties are missing ids which link them to the proper `<label>` tags.

**Screenshots**

<img width="837" height="669" alt="Image" src="https://github.com/user-attachments/assets/aba4d8a2-c05a-4112-b975-487d33ab5e9a" />

**Environment (please complete the following information):**
 - OS: linux mint
 - Browser firefox
 - XWiki version 18.1.0
 - App version 3.10.2
